### PR TITLE
Update libribasim environment

### PR DESCRIPTION
**Pull request by Hofer-Julian**
- Add PackageCompiler as dependency
- Add `Manifest.toml`

### CODE REVIEW

**Comment by visr** [comment]
This is a bit confusing, but libribasim is the environment that we want to build. PackageCompiler is the tool we want to use to build it. Therefore it shouldn't be in the libribasim environment. There should probably be a new environmentin `create_lib`, just like `create_app` in https://github.com/Deltares/Ribasim.jl/tree/main/compile/ribasim_cli. That would include PackageCompiler. There is also a precompile file there that runs some Ribasim code. Because of this I also add Ribasim to the `create_*` projects, though perhaps it's good to check if it can do without, perhaps PackageCompiler already makes sure to execute it under the `libribasim` environment.

**Comment by visr** [comment]
I also updated the manifest, since #59 changed the Ribasim dependencies a lot, and that wasn't reflected here yet.

**Comment by Hofer-Julian** [comment]
Thanks for the clarification @visr 

> Because of this I also add Ribasim to the create_* projects, though perhaps it's good to check if it can do without, perhaps PackageCompiler already makes sure to execute it under the libribasim environment.

Seems like it also works without depending on Ribasim in the `create_*` environment.

File: run/run.toml [hunk 1]
```diff
@@ -28,58 +28,3 @@ node = "../data/lhm/node.arrow"
 edge = "../data/lhm/edge.arrow"
 
 waterbalance = "../data/lhm/waterbalance.arrow"
```
Comment (Hofer-Julian) [inline run/run.toml:30]: Precompile depends on `run`, but that one seems outdated.
The ribasim artifacts don't contain a `forcing-long.arrow` or `waterbalance.arrow`.
https://github.com/visr/ribasim-artifacts/releases/tag/v0.1.0

Comment (visr) [reply run/run.toml:30]: Ah sorry about that. Forcing long is the old long format, so you can just rename it back to forcing.arrow.

**Comment by visr** [comment]
Good to know Ribasim isn't needed in the create lib env!

File: run/run.toml (hunk 1 above)
Comment (Hofer-Julian) [reply run/run.toml:30]: What should I do with waterbalance? :)

Comment (visr) [reply run/run.toml:30]: Oh that was output, that shouldn't be an issue.

Comment (Hofer-Julian) [reply run/run.toml:30]: Ah I see, thanks :)

**Comment by Hofer-Julian** [comment]
The PR should be fine to merge now.

**Review by visr — APPROVED** [review]

**Pull request merged**